server/db/versions/010_add_updated_time_field_postgresql_downgrade.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE projects DROP COLUMN updated_time;
```

Comment:
This is not very nice: that means you'll remove the updated time for all existing projects. Also, I don't get why this change is applied at all, what was the reason for that?